docs-2.0/synchronization-and-migration/replication-between-clusters.md

```diff
@@ -0,0 +1,372 @@
+# Synchronize between two clusters
+
+Nebula Graph supports data synchronization from a primary to a secondary cluster in almost real-time. It applies to scenarios such as disaster recovery and load balancing, and helps reduce the risk of data loss and enhance data security.
+
+!!! enterpriseonly
+
+    This feature applies to the Enterprise Edition only.
+
+## Synchronization workflow
+
+The synchronization works as follows:
+
+![replication between clusters](https://docs-cdn.nebula-graph.com.cn/figures/replication-between-clusters.png)
+
+1. The primary cluster sends any data written into it to the Meta listener or the Storage listener.
+2. The listener sends the data to the drainer.
+3. The drainer sends the data to the partitions of the secondary cluster through the Meta client or the Storage client.
+
+## Applicable Scenarios
+
+- Remote disaster recovery: Data synchronization enables cross-room or cross-city disaster recovery.
+
+- Data migration: The migration can be implemented by synchronizing data and then switching cluster roles, without stopping the service.
+
+- Read/Write splitting: Enable only writing on the primary cluster and only reading on the secondary cluster to lower the system load, and improve stability and usability.
+
+## Precautions
+
+- The synchronization bases on graph spaces, i.e., from one graph space in the primary cluster to another in the secondary cluster.
```

Comment:
The verb is to base something on (or upon) so we can say, for example, users can base the synchronization on graph spaces.

or use the passive form, for example, the synchronization is based on xx